client.go

```diff
@@ -282,42 +282,49 @@ func (client *Client) Close() error {
 
 // SetNetwork replaces all nodes in the Client with a new set of nodes.
 // (e.g. for an Address Book update).
-func (client *Client) SetNetwork(network map[string]AccountID) error {
-	return client.network.SetNetwork(network)
+func (client *Client) SetNetwork(network map[string]AccountID) (*Client, error) {
```

Comment:
Since this method already returned _something_ we can't change the return type :(
You can only update the return type of methods that returned nothing before.